src/MiNET/MiNET/Player.cs

```diff
@@ -1670,14 +1799,14 @@ public virtual void SpawnLevel(Level toLevel, PlayerLocation spawnPoint, bool us
 
 					SetNoAi(oldNoAi);
 
-					ForcedSendChunks(() =>
-					{
+					//ForcedSendChunks(() =>
```

Comment:
Is there a reason for not doing this anymore?

Reply:
Chunks are often sent before the client can process them, because of this they are lost.
The most correct solution is to send them only after teleportation. After that, they will not go, since MINET thinks that the player has already received them.